apps/api/src/build-openapi.ts

```diff
@@ -0,0 +1,37 @@
+// This is a very hack way to generate the swagger
+// without needing any of the service dependencies.
+// At some point we should find a better way, but
+// there might not be one.
+
+import '@frequency-chain/api-augment';
+import * as fs from 'fs';
+import { NestFactory } from '@nestjs/core';
+
+// Mock out required env vars before the module loads
+process.env.REDIS_URL = 'http://127.0.0.1';
+process.env.FREQUENCY_URL = 'http://127.0.0.1';
+process.env.PROVIDER_ACCOUNT_SEED_PHRASE =
+  'offer debate skin describe light badge fish turtle actual inject struggle border';
+process.env.PROVIDER_ID = '0';
+process.env.PROVIDER_BASE_URL = 'http://127.0.0.1';
+process.env.CAPACITY_LIMIT = '{"type":"amount","value":0}';
```

Comment:
This works, although another approach would be to simply commit an environment file specifically for generating the Swagger and sourcing it in the package script for the command that runs this.

Reply:
I thought about that, but wanted it clearly connected to it, so went this path. I didn't feel strongly about it, so if someone changes it in the future, that's fine by me.